Quorum/apis/git_api/git_manager.py

```diff
@@ -36,22 +36,16 @@ def _load_repos_from_file(self, gt_config: dict[str, any]) -> tuple[dict[str, st
         Load repository URLs from the JSON file for the given customer.
 
         Args:
-            gt_config (dict[str, any]): The ground truth configuration data.
+            gt_config (dict[str, any]): The ground truth configuration data for the customer.
 
         Returns:
             tuple[dict[str, str], dict[str, str]]: 2 dictionaries mapping repository names to their URLs.
                 The first dictionary contains the repos to diff against. The second dictionary is the verification repo.
         """
-        # Normalize the customer name to handle case differences
-        normalized_customer = self.customer.lower()
-        customer_repos = next((repos for key, repos in gt_config.items() if key.lower() == normalized_customer), None)
-        if customer_repos is None:
-            return {}, {}
```

Comment:
Why did we remove this?

Reply:
That was a bug that prevented from the git repo to update